libraries/csharp-net22-runtime/context/Pipe/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Pipe
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                throw new Exception(
+                    "no algorithm directory argument found. Please provide the path to a valid C# algorithm.");
+            }
+            string sysPath = args[0];
+            Config config = new Config(sysPath);
+            Module algoModule;
+            try
+            {
+                algoModule = new Module(config);
+            }
+            catch (Exception e)
+            {
+                ExceptionResponse response = new ExceptionResponse(e);
+                Write.WriteJsonToPipe(response);
+                Console.Out.WriteLine("PIPE_INIT_FAILED");

```

Comment:
I feel like this is a good thing to add? I can remove if it would compilcate things ofc.

Reply:
I think that will break json parsing of the output?

Reply:
I thought stdout was used for logging, where `/tmp/algoout` was the primary output response destination, no? I'm kind of flying blind with this so the smart thing to do would be to remove this.

Reply:
it doesn't add anything, and nothing will be trying to parse this so i think it should just be left out.

Reply:
ok I'll remove.